### DISCUSSION TITLE
Azure Docker 環境下 auth0 CallBack URL 會抓到 0.0.0.0:3210 [Bug]

### DISCUSSION THREAD

**Discussion by Sen-CaPoo** [lobehub/lobehub#1336] [Self Hosting] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Docker

### 🌐 Browser

Chrome

### 🐛 Bug Description

正確依照指南設定好 auth0 的相關參數，但在 LobeChat 介面 [通用設置] > [登錄帳號] 後，跳轉到 Auth0 登入頁面，輸入正確的登入資訊後，要執行 CallBack 時，會導向 0.0.0.0:3210(https://0.0.0.0:3210/api/auth/callback/auth0?code=OmJnwvJxvuoQ_nltI39UA-hopISDg2hv123RhgnFxQxxT)。

正確的情況下 [0.0.0.0:3210] 應該要是登入頁的 Domain 才對。（我這邊的 Domain 連 Port 都不需要指定，是標準 https 443）

已確認 Auth0 網站的 applications 設定正確無誤。
已確認 LobeChat 環境變數設定正確。

[a comment by an automated account is not shown]

**Comment by cy948** [comment, marked as answer]
需要增加`NEXTAUTH_URL`这个环境变量，示例见 https://github.com/lobehub/lobe-chat/blob/2b3e457ffea9637d45ede95723e697cdb54b1570/docs/self-hosting/environment-variables/basic.zh-CN.mdx#L79

[a comment by an automated account is not shown]

**Comment by Sen-CaPoo** [comment]
> 需要增加`NEXTAUTH_URL`这个环境变量，示例见
> 
> https://github.com/lobehub/lobe-chat/blob/2b3e457ffea9637d45ede95723e697cdb54b1570/docs/self-hosting/environment-variables/basic.zh-CN.mdx#L79

感謝，增加後可以正確執行了，沒注意有這個環境變數需要設定。

[a comment by an automated account is not shown]
